test/Dockerfile

```diff
@@ -1,22 +1,18 @@
-FROM python:3.5.2
+FROM hashicorp/terraform as terraform
 
-ENV TERRAFORM_VERSION 0.11.8
+FROM python:3.6
```

Comment:
Let's bump this to 3.8 unless there is a good reason. It worked fine when I reran it with 3.8.

Reply:
Check.